### PR TITLE
PT-424 | Enrolment summary

### PR DESCRIPTION
**Pull request by quyenlq**
Added an endpoint to query enrolments by staff organisation id and status. 
Added `count` field to the connection

### CODE REVIEW

File: occurrences/tests/test_api.py [hunk 1]
```diff
@@ -1774,3 +1775,75 @@ def test_cancel_occurrence(snapshot, staff_api_client, occurrence):
...
+    assert_permission_denied(executed)
+
+
+def test_enrolments_summary(
+    snapshot, staff_api_client, occurrence, mock_get_event_data
+):
+    organisation_gid = to_global_id(
+        "OrganisationNode", occurrence.p_event.organisation.id
+    )
+    EnrolmentFactory(study_group=StudyGroupFactory(), occurrence=occurrence)
```
Comment (tuomas777) [inline occurrences/tests/test_api.py:1821]: Just a minor thing of course, but using `StudyGroupFactory` here seems unnecessary as that should happen by default.

File: occurrences/schema.py [hunk 2]
```diff
@@ -372,13 +376,25 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
...
 
 
+class EnrolmentConnectionWithCount(Connection):
+    class Meta:
+        abstract = True
+
+    total_count = graphene.Int()
+
+    def resolve_total_count(root, info, **kwargs):
+        return root.length
```
[the remark this answers is not in the document]
Comment (quyenlq) [reply occurrences/schema.py:386]: That's totally reasonable, good to know how it goes in another project :D

File: occurrences/tests/test_api.py (hunk 1 above)
Comment (quyenlq) [reply occurrences/tests/test_api.py:1821]: Fixed, it was `Enrolment.objects.create` before I realized there is already EnrolmentFactory. Then I didn't bother to refactor the argument xD

**Review by tuomas777 — APPROVED** [review]
:shipit:

**Pull request merged**